lib/imgix/path.rb

```diff
@@ -84,7 +84,7 @@ def method_missing(method, *args, &block)
       end
     end
 
-    def to_srcset(width_tolerance: DEFAULT_WIDTH_TOLERANCE, **params)
+    def to_srcset(width_tolerance: DEFAULT_WIDTH_TOLERANCE, min_width: MIN_WIDTH, max_width: MAX_WIDTH, **params)
```

Comment:
I hope this doesn't cause any confusion as it is pretty similar to the imgix parameter `min-x`/`max-h`. I am basing the naming off of what is currently supported in imgix-rails but I am open to renaming things as right now is a good time to do so if we so choose.

Reply:
Hmm, that's a great point. How about `srcset_max` and `srcset_min`?